### ISSUE TITLE
Extending dial with support for conversions

### ISSUE THREAD

**Issue by asmodeus812** [monaqa/dial.nvim#32]
Hi, I was just exploring this great plugin, which does an amazing job at extending the base inc/dec features of vim, and i was reminded of a vim based plugin called vim-radical (https://github.com/glts/vim-radical) which allows the user to convert the underlying number under the cursor into another representation. To me that seems like an interesting idea. Can be useful for things like dates or time formats - converting from one date/time format to another, numbers - same idea as radical (convert decimal to hex, binary or octal etc). Or even allowing a custom conversion from one type to another, which can be defined by the user. It seems that dial already has the internal support to understand the text objects it works with anyway (dates, numbers etc.), is that a feasible improvement ?

**Comment by matu3ba** [comment]
I disagree and think that it is better to keep the plugin minimal and use established cli tools instead.

* Nevertheless, there should be support for selection of a complete number as it should be part of the parsing code anyway.
* Linking a good or multiple tools to either 1. do the conversion within neovim with some lua plugin or 2. suggest possible cli tools sounds like good things to do.
* You could even support only parsing as a function for doing stuff with external cli tooling
